docs/source/analyst/autoflow.md

```diff
@@ -0,0 +1,176 @@
+    - The name of an 'automatic' workflow parameter (i.e. a parameter that will automatically be passed by the available dates sensor). There are two 'automatic' parameters:  
+        - `reference_date`: The date of available CDR data for which the workflow is running (as an iso-format date string).  
+        - `date_ranges`: A list of pairs of iso-format date strings, describing the date ranges defined by the `date_stencil` (see the section on [date stencils](#date-stencils) below) for `reference_date`.  
+    - `flowapi_url`: The URL at which FlowAPI can be accessed. The value of this parameter will be the URL set as the environment variable `FLOWAPI_URL` inside the container.  
+- `output` (optional): Set `output: {format: pdf}` to convert this notebook to PDF after execution. Omit the `output` key to skip converting this notebook to PDF. Optionally, a custom template can be used when converting the notebook to asciidoc by setting `output: {format: pdf, template: custom_asciidoc_template.tpl}` (where `custom_asciidoc_template.tpl` should be a file in the same directory as `workflows.yml`).  
+
+#### Date stencils
+
+The default behaviour is to run a workflow for every date of available CDR data. If the notebooks refer to date periods other than the reference date for which the workflow is running, further filtering is required to ensure that all required dates are available. This can be done by providing a date stencil.
+
+Each element of the list of workflows defined in the `available_dates_sensor` block in `workflows.yml` accepts a `date_stencil` parameter, which defines a pattern of dates relative to each available CDR date. The workflow will only run for a particular date if all dates contained in the date stencil for that date are available. The date stencil is a list of elements, each of which is one of:
+
+- an absolute date (e.g.`2016-01-01`);  
+- an integer representing an offset (in days) from a reference date. For example, `-1` would refer to the day before a reference date, so a workflow containing `-1` would only run for date `2016-01-03` if date `2016-01-02` was available;  
+- a pair of dates/offsets representing a date interval (inclusive of both limits) for which all dates must be available. Absolute dates and offsets can be mixed (e.g. `[2016-01-01, -3]`).  
+
+If the `date_stencil` parameter is not provided, the default value is `[0]` (i.e. only the reference date itself needs to be available).
+
+For each date for which the workflow runs, the corresponding date ranges described by the date stencil are available to notebooks as the parameter `date_ranges`, which will contain a list of pairs of absolute dates. Note that `date_ranges` will always contain pairs of dates, even when the date stencil contains individual dates/offsets.
+
+As an example, if the date stencil is
+```yaml
```

Comment:
As written this looks like a rendering error to me, can it be more evidently yaml-formatted (e.g. with the `date_stencil` key present)?